<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnumParm</class>
 <widget class="QWidget" name="EnumParm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="__lo">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
